src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -1097,4 +1104,45 @@ public bool IsCreatedFromValidFile
             get { return isCreatedFromValidFile; }
         }
     }
+
+    public class DynamoPlayerFolderItem
```

Comment:
thanks for the comments on the members, please add a ///summary for the class as well.